skietbaan_fe/src/reducers/postReducer.js

```diff
@@ -0,0 +1,28 @@
+import {FETCH_POSTS, NEW_POST} from '../actions/types';
+
+const initialState = {
+    items: [],
+    item: {}
+
+}
+
+export default function(state = initialState, action){
+    switch(action.type){
+        case FETCH_POSTS:
+        console.log('reducer');
+            return {
+                ...state,
+                items: action.payload
+            };
+            
+         case NEW_POST:
+            return{
```

Comment:
Add space between `return` and `{`. It's inconsistent with the rest of the code

Reply:
Sorted. Kindly review